index.html

```diff
@@ -278,6 +278,29 @@ <h3 class="skill-title">
           </ul>
         </section>
       </main>
+      <footer class="footer-section">
+        <h3 class="footer-header">
+          Contact me
+        </h3>
+        <p class="footer-info">
+          If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+        </p>
+        <form action="https://formspree.io/f/xeqvkzra" class="footer-form" method="POST">
+          <ul>
+            <li class="first">
+              <input type="text" name="username" placeholder="Yeremias" value="Yeremias" maxlength="30" required>
+            </li>
+            <li class="second">
+              <input type="email" name="email" placeholder="natajayanj14@gmail.com" value="natajayanj14@gmail.com" required>
+            </li>
+            <li class="third">
+              <textarea name="comment" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
+            </li>
+          </ul>
+          <a href="#" class="btn4">Get in touch</a>
```

Comment:
I suggest you wrap the C.T.A with a `button` tag or an input element with type submit for this is good practice, Read [This](https://www.w3schools.com/tags/att_button_form.asp) for more information on how form buttons are created

Reply:
Done

Reply:
This was successfully addressed.